en_US/deploy/install.md

```diff
@@ -1,23 +1,27 @@
 # Installation
 
-EMQX binary packages are released on below operating systems:
+EMQX binary packages are released on the below operating systems:
 
 + EL7 (RedHat 7, CentOS 7)
 + EL8 (RedHat 8, RockyLinux 8, AmazonLinux 2022)
 + Raspbian 10
 + Debian 9
 + Debian 10
++ Debian 11
 + Ubuntu 16.04
 + Ubuntu 18.04
 + Ubuntu 20.04
 + macOS 10
```

Comment:
Not anymore :speak_no_evil:

Reply:
🤔 https://github.com/emqx/emqx/releases/tag/v5.0.3 here we have Debian 11 builds

Reply:
The comment was for `macOS 10`, which was removed [here](https://github.com/emqx/emqx/pull/8542)

Reply:
Oh, thanks. Was moked by the interface.

Reply:
✅